cli/server.ts

```diff
@@ -0,0 +1,46 @@
+import express from 'express'
+import debug from 'debug'
+import cors from 'cors'
+import bodyParser from 'body-parser'
+import namespace from '@rdfjs/namespace'
+import * as transform from './lib/commands/transform'
+
+const ns = {
+  pipeline: namespace('urn:pipeline:cube-creator'),
+}
+
+const pipelines = {
+  TransformFiles: ns.pipeline('#Main'),
+}
+
+async function main() {
+  const log = debug('cube-creator')
+
+  const app = express()
+
+  app.enable('trust proxy')
+
+  app.use(cors())
+
+  app.use(bodyParser.urlencoded())
```

Comment:
We'll need RDF support here. Please use [@rdfjs/express-handler](https://github.com/rdfjs-base/express-handler) for reading the request body as RDF

Reply:
Do we? We send one argument here and in the old pipeline there was only a querystring.

Reply:
Sorry so maybe I misread. Are you sending the job URI as query string?

Reply:
I expressed myself badly
```typescript
  const form = new URLSearchParams({
    JOB_URI: job.value,
  })

  const res = await fetch(pipelineURI, {
    method: 'POST',
    body: form,
  })
```

Reply:
OK, case closed